<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rede</class>
 <widget class="QWidget" name="Rede">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="valor_local">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="txt_local">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Rede Local:</string>
   </property>
  </widget>
  <widget class="QLabel" name="valor_internet">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="txt_internet">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Internet:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_ip_local">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>IP Local:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_ip_externo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>IP Local:</string>
   </property>
  </widget>
  <widget class="QLabel" name="valor_ip_local">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="valor_ip_externo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>buscando...</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="valor_endereco">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>581</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>www.google.com.br</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="check_ping">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ping</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="check_traceroute">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Traceroute</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_testar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Testar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="valor_resposta">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>581</width>
     <height>151</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
